src/single_instruction_test/heap.rs

```diff
@@ -34,6 +34,12 @@ impl HeapInterface for Heap {
         // This is wrong, but this method is only used to get the final return value.
         vec![]
     }
+
+    fn memset(&mut self, src: &[U256]) {
+        for (i, word) in src.iter().enumerate() {
+            self.write_u256((i * 32) as u32, *word);
```

Comment:
This would break if the mock decommit returned more than one u256 of code page. But maybe it is ok like this.